src/ducks/balance/Balance.jsx

```diff
@@ -175,8 +178,13 @@ class Balance extends PureComponent {
     }
   }
 
-  fetchTriggers() {
-    const client = this.props.client
+  async fetchTriggers() {
+    const { client } = this.props
+    if (__TARGET__ === 'mobile') {
+      const pouchManager = client.links[0].pouches
+      pouchManager.stopReplicationLoop()
+      await pouchManager.startReplicationLoop()
```

Comment:
Is this the final solution or just a temporary one? It doesn't seem good to rely on pouch link internals. Shouldn't the link expose a method to do this?

Reply:
Done in https://github.com/cozy/cozy-client/pull/434